web/controllers/game_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Yggdrasil.GameController do
+  use Yggdrasil.Web, :controller
+
+  alias Yggdrasil.Game
+
+  def index(conn, _params) do
+    games = Repo.all Game
```

Comment:
Btw I'm also not in favor of this any more

```Elixir
Yggdrasil.Game.list_games()
Yggdrasil.Game.get_game(123)
Yggdrasil.Game.list_characters()
Yggdrasil.Game.get_character(123)
Yggdrasil.Game.add_character(...)
```

What ever this turns into, I think it's ok to have several modules that are composed instead of one like this. If we determine a need to put a process around character creation for example `Yggdrasil.Character.create()` or something along those lines but point being that I think having task specific modules are probably better than one module where the world of api calls hangs off of.